File: Chapter13/SwiftExample/SwiftExample/ObjectiveCFiles/MessageBuilder.m
//
//  MessageBuilder.m
//  SwiftExample
//
//  Created by Jon Hoffman on 7/15/17.
//  Copyright © 2017 Jon Hoffman. All rights reserved.
//

#import "MessageBuilder.h"
#import "SwiftExample-Swift.h"

@implementation MessageBuilder

-(NSString *)getPersonalizedMessageForName:(NSString*)name {
    Messages *messages = [[Messages alloc] init];
    
    NSString *retString = [NSString stringWithFormat:@"To: %@, %@",name, [messages getMessage]];
    return retString;
}
@end

File: Chapter13/ObjectiveCExample/ObjectiveCExample/SwiftFiles/MessageBuilder.swift
//
//  MessageBuilder.swift
//  ObjectiveCExample
//
//  Created by Jon Hoffman on 7/15/17.
//  Copyright © 2017 Jon Hoffman. All rights reserved.
//

import Foundation

class MessageBuilder: NSObject {
    
    func getPersonalizedMessage(name: String) -> String {
        let messages = Messages()
        let retMessage = "To: " + name + ", " + messages.getMessage()
        return retMessage;
    }

}

